disnake/guild.py

```diff
@@ -847,7 +850,11 @@ def public_updates_channel(self) -> Optional[TextChannel]:
 
     @property
     def emoji_limit(self) -> int:
-        """:class:`int`: The maximum number of emoji slots this guild has."""
+        """:class:`int`: The maximum number of emoji slots this guild has.
+
+        Premium emojis (i.e. those associated with subscription roles) count towards a
+        separate limit of 25.
```

Comment:
thoughts on a possible `premium_emoji_limit` property of 25?

not sure if that's a good idea or not tbh

Reply:
if a nitro-dependent number of premium emoji slots gets added, that would make sense, yeah. But I don't think we need one for a constant value just yet.